lib/config/autoconfig.js

```diff
@@ -10,12 +10,13 @@
 //------------------------------------------------------------------------------
 
 const lodash = require("lodash"),
-    eslint = require("../eslint"),
+    Linter = require("../Linter"),
     configRule = require("./config-rule"),
     ConfigOps = require("./config-ops"),
     recConfig = require("../../conf/eslint-recommended");
 
 const debug = require("debug")("eslint:autoconfig");
+const eslint = new Linter();
```

Comment:
Nitpick: I think you missed this variable when renaming `eslint` variables to `linter`.